src/util/validationUtils.js

```diff
@@ -169,9 +169,30 @@ const checkNoUnsupportedParams = (query, unsupportedParams, operationName) => {
   }
 };
 
+/**
+ * Pulls query parameters from both the body and creates a new parameters map
+ * @param {Object} query the query terms on the request URL
+ * @param {Object} body a FHIR Parameters object passed into the body of the request passed in
+ * @returns {Object} an object containing a combination of request parameters from both sources
+ */
+const gatherParams = (query, body) => {
+  const params = { ...query };
+
+  if (body.parameter) {
+    body.parameter.reduce((acc, e) => {
+      if (!e.resource) {
+        acc[e.name] = e.valueDate || e.valueString || e.valueId || e.valueCode;
```

Comment:
Is there a better way to accomplish this? Not sure how to specify the expected key for given params here.

Reply:
I think that for now this approach is good since these are the only value[x] keys that need to be supported for the care gaps Parameters object, although that may restrict this function to only being valid for the care gaps operation. In the future, maybe we can add a function to determine what value[x] keys we need to support for the given operation, and then we can pass those value[x] keys into the `gatherParams()` function. I think this would be out of scope for now.

Reply:
I do think that we should add a comment above this line explaining that these are the only keys that we need to support for now